### PR TITLE
Update nugets

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-modules/issues/232 https://github.com/neo-project/neo/issues/1245

### CODE REVIEW

**Comment by superboyiii** [comment]
> Close #232 [neo-project/neo#1245](https://github.com/neo-project/neo/issues/1245)

Maybe we should upgrade nuget to 3.0.0-CI00907, since neo-cli's dependency now is changed to that. And since the incompatible change here: [3.0.0-CI00906](https://github.com/neo-project/neo/blob/946fb73d01ae942635570430b550c5a41f747654/src/neo/Plugins/ILogPlugin.cs#L5),`message` type should be changed to `object` in `void ILogPlugin.Log(...)` of SystemLog.
It works well on my local after these are applied,.
image

**Comment by shargon** [comment]
@superboyiii please, could you test it again ?

File: src/SystemLog/Logger.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ protected override void Configure()
             Settings.Load(GetConfiguration());
         }
 
-        void ILogPlugin.Log(string source, LogLevel level, string message)
+        void ILogPlugin.Log(string source, LogLevel level, object message)
```
Comment (erikzhang) [inline src/SystemLog/Logger.cs:17]: Need to consider whether the type of `message` is `Exception` or other.

Comment (shargon) [reply src/SystemLog/Logger.cs:17]: What should be the behaviour when it's an Exception?

Comment (erikzhang) [reply src/SystemLog/Logger.cs:17]: https://github.com/neo-project/neo-node/commit/82033b46d36a69b31e24e6f421e5e174a2ca0d88#diff-2ad153cafa6b784ceb93f0f6a668bcbdL31-L49

**Comment by superboyiii** [comment]
@shargon Find a issue on `RpcBlockHeader.FromJson`.
https://github.com/neo-project/neo-modules/issues/238

**Comment by shargon** [comment]
@chenquanyu could you help me with the RPC ut?

**Comment by joeqian10** [comment]
@shargon @chenquanyu I will look into this.

**Comment by shargon** [comment]
@superboyiii could you check it again please?

**Comment by superboyiii** [comment]
@shargon I'll check it soon.

**Comment by superboyiii** [comment]
@shargon I've tested all RpcClient mehods related to modified parts, like `GetBlockHeader`, `GetBlock`, `GetContractState`, they all works well, return the same response as RPC invocation result. And SystemLogs also works well.

**Pull request merged**